src/pages/news/2021-04-26-OSOR-interview.en.md

```diff
@@ -0,0 +1,15 @@
+---
+path: "/en/news/2021-04-26-OSOR-interview"
+date: "2021-04-26"
+title: "OpenZaak featured in the European Open Source Observatory (OSOR)"
+lang: en
+template: default
+---
+
+# OpenZaak featured in the European Open Source Observatory (OSOR)
+
+The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published on April 7th an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak.
```

Comment:
```suggestion
The European portal [OSOR](https://joinup.ec.europa.eu/collection/open-source-observatory-osor), Open Source Observatory, published an interview with Lazo Bozarov, Senior Supply Manager Digital Services at Utrecht Municipality about OpenZaak on April 7th.
```

Reply:
Does it really matter?

Reply:
Not semantically, but if we follow the journalistic method of leading with the most important message, I would say the date is the least important piece of information in this sentence.